src/Components/Endpoints/src/RazorComponentEndpointHost.cs

```diff
@@ -19,7 +19,6 @@ internal class RazorComponentEndpointHost : IComponent
 {
     private RenderHandle _renderHandle;
 
-    [Parameter] public IComponentRenderMode? RenderMode { get; set; }
```

Comment:
This is some unrelated cleanup. This component type is `internal` and we were not actually using this parameter, and it wouldn't be valid to have a root-level rendermode anyway, so it doesn't make sense to leave this vestigial code in place.